projects/dotcms-field-elements/src/utils/errorHandling.tsx

```diff
@@ -0,0 +1,62 @@
+const fieldValidationMap = {
+    options: stringValidator,
+    regexCheck: regexValidator
+};
+
+interface PropValidationInfo<T> {
+    field: {
+        type: string;
+        name: string;
+    };
+    name: string;
+    value: T;
+}
+
+class DotFieldParamError<T> extends Error {
```

Comment:
Should we move this class to it owns file?

Reply:
at this moment this is being only used here